src/VisualStudio/Core/Test/Snippets/VisualBasicSnippetExpansionClientTests.vb

```diff
@@ -424,7 +428,8 @@ End Class</Test>
                     Guids.CSharpLanguageServiceId,
                     surfaceBufferDocument.GetTextView(),
                     subjectBufferDocument.GetTextBuffer(),
-                    Nothing)
+                    Nothing,
+                    workspace.ExportProvider.GetExports(Of ArgumentProvider, OrderableLanguageMetadata)())
 
                 SnippetExpansionClientTestsHelper.TestProjectionBuffer(snippetExpansionClient, surfaceBufferDocument, expectedSurfaceBuffer)
             End Using
```

Comment:
can we have some actual tests, and ideally one integration test?

Reply:
➡️ Added a set of integration tests showing some key paths through the UX

Reply:
👍